x-pack/plugins/maps/common/descriptor_types/layer_descriptor_types.ts

```diff
@@ -71,6 +71,7 @@ export type LayerDescriptor = {
   style?: StyleDescriptor | null;
   query?: Query;
   includeInFitToBounds?: boolean;
+  showTooltips?: boolean;
```

Comment:
Can you move to VectorLayerDescriptor since only vector layers support tooltips?